libs/ledger-live-common/src/exchange/swap/getExchangeRates.ts

```diff
@@ -115,6 +156,25 @@ const getExchangeRates: GetExchangeRates = async (
       };
     }
   });
+  if (decentralizedSwapAvailable()) {
+    // TODO fill dexProvider list
+    // const dexProviders = ["paraswap", "oneinch"];
+    const dexProviders = [];
+    dexProviders.filter((dexProvider) => {
+      if (rates.findIndex((rate) => rate.provider === dexProvider) < 0) {
```

Comment:
@sarneijim the response contains `providerType` field (`CEX` or `DEX`), I think you can use it to avoid hardcoding the providers

Reply:
rates will not contain "DEX" providers for those token that are not supported by back-end.

1. Front-end use pair call to get what "providers" are supported by back-end for some specific currency pair.
2. Back-end  answers with those provider only.

Reply:
ha yes you're right! my bad nevermind